common/locales/en/groups.json

```diff
@@ -177,6 +177,7 @@
     "inviteMissingEmail": "Missing email address in invite.",
     "partyMustbePrivate": "Parties must be private",
     "userAlreadyInGroup": "User already in that group.",
+    "cannotInviteSelfToGroup": "You cannot invite yourself to a group.",
```

Comment:
We _could_ use the `userAlreadyInGroup` string, but I think being more specific is probably better.